### PR TITLE
Modify field Trial flow chart and edits to RC2 Chptr 4 & 5

### PR DESCRIPTION
**Pull request by steelescotr**
docs\gov\chapter 9
Modifies the Field Trial graphic to replace CNTT with Anuket

### CODE REVIEW

**Review by pgoyal01 — APPROVED** [review]

**Comment by gkunz** [comment]
Hi @steelescotr. Thanks! I noticed that new new png file does not have a transparent background, but a black one. I suggest exporting it again with a transparent background.

Also, I think we should absolutely upload the "editable sources" of the figures. Do you have those available, too?

Finally, I don't think we need a copy of the old file. Git will keep a copy for us.

**Review by CsatariGergely — CHANGES_REQUESTED** [review]
- We do not need field_trials_old.png, it is stored in git history. Please delete it. 
- Please add the editable version of the file with the name field_trails.pptx

**Comment by steelescotr** [comment]
I have updated the Background color on field_trials.png, deleted the .old file, and created new .png files for nfvi_transition and vnf_cnf_tranistion. I added the pptx files under artefacts.

**Comment by gkunz** [comment]
Hi @steelescotr. Thanks for the update. The background is now white instead of transparent, but that has been the case for the other two figures as well. I don't mind a white background, so that's ok for me.

**Review by gkunz — APPROVED** [review]

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:213]:
```suggestion
3. Engaged vendors successfully configure their VI and run the RC1 or RC2 test suite and are able to provide expert feedback
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Review by rgstori — APPROVED** [review]

File: doc/gov/chapters/chapter09.rst
Comment (steelescotr) [reply doc/gov/chapters/chapter09.rst:213]: Will remove the dashes.

**Comment by steelescotr** [comment]
@CsatariGergely Lease review requested change and approve if you are satisfied with the it.

**Review by CsatariGergely — APPROVED** [review]

**Comment by steelescotr** [comment]
@walterkozlowski Can you please add your review to this PR?

**Comment by steelescotr** [comment]
Thanks for the recommendations Georg.....I will update the graphic, and remove the old file.....
As for the editable source, yes I do have that...it's a pptx file. Will add it to the figures directory.


Scot

805-825-1100

**Comment by steelescotr** [comment]
Modifications made to the Chapter 4 - included all of the workload tests defined in RA2. 
Chapter 5 updated with references to resources.

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:292]:
```suggestion
#. Engaged vendors successfully configure their Cloud Infrastructure and run the RC1 or RC2 test suite and are able to provide expert feedback
```

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:307]:
```suggestion
-  RA1 or RA2 Specifications
```

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:308]:
```suggestion
-  Cloud Infrastructure implementation support methods ( i.e. cookbooks, installation manuals, how to guides etc.)
```

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:431]:
```suggestion
-  Cloud Infrastructure Implementation methods and procedures (cookbook, etc)
```

File: doc/gov/chapters/chapter09.rst
Comment (CsatariGergely) [inline doc/gov/chapters/chapter09.rst:432]:
```suggestion
-  RA1 or RA2 Specifications
```

**Review by CsatariGergely — CHANGES_REQUESTED** [review]

**Review by walterkozlowski — APPROVED** [review]
There are still open change proposals from @CsatariGergely which make sense to me.  I am approving in understanding that these will be taken care of.

**Comment by steelescotr** [comment]
@CsatariGergely @walterkozlowski, I have addressed the requested changes. please let me know if there are other issues to resolve.

**Review by rgstori — APPROVED** [review]

**Review by CsatariGergely — APPROVED** [review]

**Comment by steelescotr** [comment]
@rgstori , @CsatariGergely , @walterkozlowski - Could one of you Merge this this PR? it has your approvals already.

**Comment by collivier** [comment]
The main RC2 contributor is missing in the review.
And there are many RST issues in it.

I'm very surprised that this patch has not been brought to TSC yet @CsatariGergely.
I think all patchset must be treated fair.
Thank you in advance to apply the same RA1 and RM treatment everywhere.

**Comment by CsatariGergely** [comment]
Can you share what rst isses you found in here? I fixed what I could found here: #3289 . Are there any others need fix?

I treat the consumability of Anuket as a priority item as without a decent landing page and documentation, misleading repo names and readthedocs url-s  we can not expect adoption of the project. All of these topics need changes in the "no mans land" not covered in sub-project meetings or projects. This is the reason why I put them to the TSC meeting agenda. If you think other pr-s should be discussed also in TSC, the agenda is open.

**Pull request merged**
